js/main.js

```diff
@@ -10,23 +10,58 @@
   };
 
   let changeTodo = (changeRow) => {
-
-    if(todos[changeRow].status === "作業中"){
-      todos[changeRow].status = "完了";
+    if(todos[changeRow].status === '作業中'){
+      todos[changeRow].status = '完了';
     }else{
-      todos[changeRow].status = "作業中";     
+      todos[changeRow].status = '作業中';     
     }
+    displayTodo(todos);
+  };
+  
+  let alltodo = document.getElementById('alltodo');
```

Comment:
再代入はしないと思うので `const` でいいと思います。
以降の類似箇所も全て同様ですね。

```suggestion
  const alltodo = document.getElementById('alltodo');
```